spec/cucumberConf.js

```diff
@@ -0,0 +1,35 @@
+var env = require('./environment');
```

Comment:
I know we do not provide first party cucumber support and there are no feature files, but users who use cucumber would have a look at the frameworks.md file in which we have mentioned a link to a sample cucumber conf.js. we need to either remove it or add a sample conf file. I would suggest let's remove it and i could add a reference link to our protractor-cookbook where users can have a look at the cucumber conf file.